### PR TITLE
updating to support binary message coming across the queue

### PR DESCRIPTION
**Pull request by rcordeau**
This change is [Reviewable](https://reviewable.io/reviews/smarterapp/rdw_ingest/303)

### CODE REVIEW

File: group-processor/src/main/java/org/opentestsystem/rdw/ingest/group/GroupProcessorConfiguration.java [hunk 1]
```diff
@@ -27,7 +29,10 @@ public GroupProcessorConfiguration(final GroupProcessor groupProcessor) {
     public void process(final Message<?> message) {
 
         final RdwMessageHeaderAccessor accessor = RdwMessageHeaderAccessor.wrap(message);
-        final String payload = (String) message.getPayload();
+        //payload should be a json object
+        final String payload = new String((byte[]) message.getPayload(), StandardCharsets.UTF_8);
+
+        logger.info("received " + accessor.getContent());
```
Comment (rcordeau) [inline group-processor/src/main/java/org/opentestsystem/rdw/ingest/group/GroupProcessorConfiguration.java]: Just temporary for now.

File: group-processor/src/test/java/org/opentestsystem/rdw/ingest/group/GroupProcessorConfigurationTest.java [hunk 2]
```diff
@@ -32,13 +32,13 @@ public void createProcessor() throws IOException {
         processor = new GroupProcessorConfiguration(groupProcessor);
 
         accessor = RdwMessageHeaderAccessor.wrap(null);
-        accessor.setContentType(MediaType.TEXT_PLAIN_VALUE);
+        accessor.setContentType(MediaType.APPLICATION_JSON);
```
Comment (malaffoon) [inline group-processor/src/test/java/org/opentestsystem/rdw/ingest/group/GroupProcessorConfigurationTest.java]: Not sure if this is just for the test but we should know that JSON != UTF-8.

Comment (rcordeau) [inline group-processor/src/test/java/org/opentestsystem/rdw/ingest/group/GroupProcessorConfigurationTest.java]: Mistake

Comment (rcordeau) [inline group-processor/src/test/java/org/opentestsystem/rdw/ingest/group/GroupProcessorConfigurationTest.java]: Wait - isn't the default encoding UTF-8?

Comment (malaffoon) [inline group-processor/src/test/java/org/opentestsystem/rdw/ingest/group/GroupProcessorConfigurationTest.java]: my mistake ... didn't used to be that way ... okay, a looong time ago it didn't used to be that way.

**Pull request merged**